conans/model/conan_file.py

```diff
@@ -115,11 +115,18 @@ def __init__(self, display_name=""):
 
     def serialize(self):
         result = {}
+
+        # Some fields are requested in the docs to be tuples, but we accept single strings too,
+        # ensure those fields are serialized as lists so things like CIs don't break unexpectedly,
+        # as it did for cci
+        def _ensure_iterable_if_needed(k, value):
```

Comment:
On a second thought, lets change them to list at origin. ``provides`` is already converted to tuple just for checking in ``check_graph_provides()`` function. 
It will be cleaner if it is accessed at some point by deployers, users navigating ``self.dependencies``, etc